### DISCUSSION TITLE
Welcome to msgraph-sdk-python Discussions!

### DISCUSSION THREAD

**Discussion by petrhollayms** [microsoftgraph/msgraph-sdk-python#682] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dominique-lambert-OBS** [comment]
Hello,
I'have a closed case [https://github.com/microsoftgraph/msgraph-sdk-python/issues/820](url) that I would like to reopen. How can I do it ?

**Reply by andrueastman** [reply]
Looks like this is re-opened now.

**Reply by dominique-lambert-OBS** [reply]
yes thanks !
